src/PoolManager.sol

```diff
@@ -307,6 +311,17 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         emit ProtocolFeeUpdated(id, newProtocolFee);
     }
 
+    function collectProtocolFees(address recipient, uint256 id, uint256 amount)
```

Comment:
There was a comment on the previous PR on whether or not we should expose this entrypoint for the feeController. I think governance would appreciate just being able to call this func on core, and not having to lock + handle the callback. What do ppl think?

Reply:
yeah i think worth keeping it as this function so its as simple as possible without needing a custom lockAcquired function

Reply:
though do you think its worth putting a `burn` function on our 6909 that anyone can call (without a lock) that just burn their 6909s and get the tokens? `burn` doesnt feel like it would need a lock as it wouldnt change any of the reserves

Reply:
Yea I like just having the collect fees func and not exposing burn outside of a lock

Reply:
I think we should expose just collect fees then?

Reply:
Not a fan of this function if we accumulate in 6909. Let the owner collect fees just like any other 6909 owner. This is basically how V2 works, but theres a utility contract so it's easy to make a governance call.

Reply:
Oh interesting, like push it to periphery? I think the main rationale was to not make governance handle lock etc. but yea that would work